src/components/menu-component.jsx

```diff
@@ -104,10 +124,27 @@ class MainMenu extends React.Component {
 
 
   render() {
+    const actions = [
```

Comment:
Could you change this variable name to `deleteNotebookDialogOptions`? `actions` are already a term understood in the Redux world.